test/wallet/money-in-sdd.js

```diff
@@ -51,6 +51,9 @@ describe('money in sdd', function () {
       });
     }).then(function (transaction) {
       return done();
-    }).catch(done);
+    }).catch((err) => {
+      console.log(err); // Url returned by lemonway on the dev environment fail to validate SSD Mandate
```

Comment:
This does not help, no?
I can't open the link, but let's keep away from this PR on iDEAL anyway